### ISSUE TITLE
Unable to select local folders to backup.

### ISSUE THREAD

**Issue by cloudymark** [jlesage/docker-cloudberry-backup#4]
When I go to select the backup folders, all I see is the below. Any ideas?

I have a folder /storage that I've mapped to the container folder /storage and have validated that this works by browsing to it via console.

![image](https://user-images.githubusercontent.com/26938147/44914056-f96f8d00-ad26-11e8-9071-f4fd6277f750.png)

**Comment by jlesage** [comment]
How did you created the container?
Could you also provide the output of the following commands:
  - `docker inspect [container name]`
  - `docker exec [container name] df -h`
  - `docker exec [container name] ls -l /`

**Comment by cloudymark** [comment]
I created it using the following:

  docker run -d \
    --name=cloudberry-backup \
    -p 5800:5800 \
    -v /docker/containers/cloudberry-backup/config:/config:rw \
    -v /storage:/storage:ro \
                -e USER_ID=1000 -e GROUP_ID=1004  \
                --restart=always \
                -e TZ="TZ=Europe/London" \
                --network=media \
    jlesage/cloudberry-backup


Outputs attached.

Thanks for helping. I have worked around it by adding a symlink into the config folder to my storage folder, but its a strange issue.

Mark.

________________________________

**Comment by cloudymark** [comment]
Apologies.
[df.txt](https://github.com/jlesage/docker-cloudberry-backup/files/2340437/df.txt)
[inspect.txt](https://github.com/jlesage/docker-cloudberry-backup/files/2340438/inspect.txt)
[ls.txt](https://github.com/jlesage/docker-cloudberry-backup/files/2340439/ls.txt)

**Comment by jlesage** [comment]
My guess is that the software doesn't display the folder because of its filesystem:

```
mergerfsPool              4.5T   1012.3G      3.3T  23% /storage
```

I will try to contact CloudBerry team to see if this can be fixed.

**Comment by cloudymark** [comment]
Well that was my assumption, but when I added a link in the config folder to storage, i am able to browse to it fine and then select it for backup.

Also, why do I not see the local files system within the docker image? Or is that by design?

**Comment by jlesage** [comment]
> Well that was my assumption, but when I added a link in the config folder to storage, i am able to browse to it fine and then select it for backup.

Yes so it's in fact just a UI bug.  The `/storage` folder is not shown even if it exists and accessible.

> Also, why do I not see the local files system within the docker image? Or is that by design?

What do you mean exactly be the "local files system" ?

**Comment by cloudymark** [comment]
The file system in the docker image itself. It doesn't even show those folders.  It just targets the config folder and some other random files.

**Comment by DaddyMacCadillac** [comment]
Any update on this issue? I have the same problem. My /storage docker volume is mapped to a ZFS pool on the host. When I start this docker image, I can see /storage and browse it and see all my files within the ZFS pool. However when trying to setup a backup, this storage folder does not exist in the UI. Even if I place a symbolic link in the config folder to /storage, it still doesn't show up. Any ideas?

**Comment by jlesage** [comment]
Unfortunately I didn't get answers yet from CloudBerry about this issue.  I pinged them again yesterday.

To document unsupported filesystems, can you post the output of `docker exec [container name] df -h`?

**Comment by sneakernuts** [comment]
Having the same issue as well all of a sudden. The GUI was still working two weeks ago, then after a reboot of the host I'm not seeing the paths anymore. However, since the configuration is still in tact, I can still perform backups using the same backup plan. Unable to make any changes to it though.

**Comment by jlesage** [comment]
Looks like an issue where the mounts on the host were not ready at the time the container was started.  Did you try to restart the container again to see if you still get the issue ?

**Comment by sneakernuts** [comment]
> Looks like an issue where the mounts on the host were not ready at the time the container was started. Did you try to restart the container again to see if you still get the issue ?

I have tried restarting and still get the issue.

**Comment by techkodeninja** [comment]
so this issue is still goin going on since then, is 2025.
